mne/viz/evoked.py

```diff
@@ -1476,15 +1500,21 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
     vlines : list of int
         A list of integers corresponding to the positions, in seconds,
         at which to plot dashed vertical lines.
-    ci : float | None
-        If not None and `evokeds` is a [list/dict] of lists, a confidence
-        interval is drawn around the individual time series. This value
-        determines the CI width. E.g., if this value is .95 (the default),
-        the 95% parametric confidence interval is drawn.
-        If None, no shaded confidence band is plotted.
+    ci : float | callable | None
+        If not None and ``evokeds`` is a [list/dict] of lists, a shaded
+        confidence interval is drawn around the individual time series. If
+        float, a percentile bootstrap method is used to estimate the confidence
+        interval and this value determines the CI width. E.g., if this value is
+        .95 (the default), the 95% parametric confidence interval is drawn. If
+        a callable, must take as its single argument an array
+        (observations x times) and return the upper and lower confidence bands.
+        If None, no confidence band is plotted.
```

Comment:
This doesn't seem to be accurate: if I use `ci=None` I get an error: `ci must be float or callable, got <class 'NoneType'>`

Reply:
@jona-sassenhagen I don't get how you can have a confidence interval when passing evoked data. What you do you bootstrap / resample if you just have the average?

Reply:
Start of the description is `If not None and ``evokeds`` is a [list/dict] of lists, `, so it must be across the innermost list

Reply:
Yes, the function gives the CI across multiple evokeds. Typically, it will be lists of subjects, but in this case (epochs image), it gives the CI across trials.

Reply:
Note to self: fix doc, it is the 95% bootstrap CI, not the parametric CI

Reply:
ok makes sense.